src/providers/UserProvider.tsx

```diff
@@ -42,6 +42,16 @@ export default function UserProvider({ children }: UserProviderProps): JSX.Eleme
         setUser(response.user!);
         if (response.user && !userState?.gtmInstrumented && (window as any).INIT_GTAG) {
           setUserState({ gtmInstrumented: true });
+
+          // Put the language in the "lang" attribute of the <html> tag before initializing Google
+          // Analytics because the cookie consent UI code will look there to determine which
+          // language to use for the consent UI. This needs to happen before the call to INIT_GTAG,
+          // so it lives here instead of in LocalizationProvider.
+          const locale = response.user.locale;
+          if (locale) {
+            document.documentElement.setAttribute('lang', locale);
+          }
```

Comment:
else `en`?

Reply:
English is the default.

Reply:
it is recommended to set the default https://www.w3.org/International/questions/qa-html-language-declarations
.. and i believe if unset the default is unknown
https://www.w3.org/TR/html401/struct/dirlang.html

Reply:
Good point. I'll send out a followup.